magit-todos.el

```diff
@@ -400,6 +419,22 @@ If PEEK is non-nil, keep focus in status buffer window."
   (interactive)
   (magit-todos-jump-to-item 'peek))
 
+(defun magit-todos-next-todo ()
```

Comment:
Rather than do the `item` checking in these functions, we could use `if-let*` in `magit-todos-jump-to-item` to check whether point is on a valid item.